ext/civicrm_admin_ui/ang/afsearchEventTemplates.aff.html

```diff
@@ -0,0 +1,7 @@
+<div af-fieldset="">
+  <fieldset class="af-container af-layout-inline af-container-style-pane af-collapsible af-collapsed" af-title="Filters">
```

Comment:
@pradpnayak could you switch this to the new format which uses `<details>` element?

Reply:
There's some recently [updated docs](https://docs.civicrm.org/dev/en/latest/framework/ui/#html-accordion) explaining: